key_for_docker

```diff
@@ -0,0 +1,42 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Why are we checking in a private key in plaintext into the repo?

Reply:
Because it's used only for ci test in the repo, right?

Reply:
Right.

Reply:
I'm of two minds here. On one hand, it does trigger a security warning, and it is possible to use it to access the docker vm running in the test server. On the other hand, generating it is one additional step people would have to do to be able to run tests. I'm inclined to add it to the `.gitignore` and mention generating it as a one-time step.
Thoughts?

Reply:
Sounds good